containers/kubernetes/reference-content/using-load-balancer-annotations.mdx

```diff
@@ -0,0 +1,96 @@
+---
+meta:
+  title: Using Scaleway Load Balancer annotations
+  description: This page explains how to use Scaleway Load Balancer annotations
+content:
+  h1: Using Scaleway Load Balancer annotations
+  paragraph: This page explains how to use Scaleway Load Balancer annotations
+tags: kubernetes load-balancer load-balancer-annotations annotations
+dates:
+  validation: 2023-10-23
+  posted: 2021-08-12
+categories:
+  - kubernetes
+---
+
+## Overview
+
+In Kubernetes, annotations are a way to attach metadata to objects, like pods, services, and more. These annotations are key-value pairs that can be useful for tools and libraries interacting with these objects. These annotations are used to influence the behavior, settings, or configurations of the provisioned load balancer resource.
+
+When you [create a Load Balancer](/containers/kubernetes/reference-content/kubernetes-load-balancer/) for your Kubernetes cluster, it will be created with a default configuration, unless you define its configuration parameters via **annotations**. Load Balancer annotations let you configure parameters such as the balancing method health check settings and more.
```

Comment:
@RoRoJ it's weird but in the staging preview I see an extra ] after balancing method.
![Capture d’écran 2023-11-06 à 17 43 47](https://github.com/scaleway/docs-content/assets/109532009/a2ec7d89-6330-4475-a08f-5481389ad753)

Reply:
The staging is not dynamically updated with the base branch - we've corrected this since I deployed the staging.